src/main/kotlin/edu/ycp/cs482/iorcapi/resolvers/ItemMutationResolver.kt

```diff
@@ -12,4 +12,9 @@ class ItemMutationResolver(
     fun addItemToVersion(name: String, description: String, price: Float,
     itemClasses: List<String>, version: String, type: ObjType)
         = itemFactory.addItem(name, description,price, itemClasses, version, type)
+
+    fun removeItem(id: String):String{
+        itemFactory.deleteItem(id)
```

Comment:
Another one. Need to deal with the consequences of trying to delete nonexistant items

Reply:
same as previous